### DISCUSSION TITLE
PDFs of revealjs: effects of the chalkboard option and an extra blank page

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#4051] [Q&A]
I noticed my PDFs (created in a browser) of revealjs previews of presentations were not taking the full page. I diagnosed this to be the `chalkboard` option. I'm not sure it needs fixing, but I figure I would warn others in case of trouble.

Also, the last page if blank. I found the `pdf-page-height-offset:` option, but setting it with values such as 1, -1, 100, -100, had no effect on the PDF (there's still a blank page at the end). Not a major problem, but just thought I'd give the feedback.

I'm using Quarto 1.3.24 (because of a few quirks in Windows with newer versions). 

Here's my MWE:

````markdown
---
format:
  revealjs:
    theme: beige
    chalkboard: true
---

# Some sample java code

```java
ArrayList a = new
    ArrayList();
```
````

Here's a screen capture of the print dialog in Chrome:

![image](https://user-images.githubusercontent.com/7606540/213933389-4e61361c-c4a9-4cf5-81be-932d036d4fb1.png)

**Comment by cscheid** [comment]
> I'm using Quarto 1.3.24 (because of a few quirks in Windows with newer versions).

I know that was not your point, but can you expand? We'd like to fix regressions before the release!

**Comment by cderv** [comment]
> Also, the last page if blank. I found the pdf-page-height-offset: option, but setting it with values such as 1, -1, 100, -100, had no effect on the PDF (there's still a blank page at the end)

Related to this https://github.com/quarto-dev/quarto-cli/issues/3997 I believe. 

> I noticed my PDFs (created in a browser) of revealjs previews of presentations were not taking the full page. I diagnosed this to be the chalkboard option. I'm not sure it needs fixing, but I figure I would warn others in case of trouble.

I wonder if this is a plugin issue or reveal.js issue; I wanted to check Chalkboard demo https://rajgoel.github.io/reveal.js-demos/chalkboard-demo.html#/ but I can't even print it

**Reply by fuhrmanator** [reply]
> I wonder if this is a plugin issue or reveal.js issue; I wanted to check Chalkboard demo https://rajgoel.github.io/reveal.js-demos/chalkboard-demo.html#/ but I can't even print it

Chalkboard breaks other features already, and Quarto reminds me when I try to preview, at least with `self-contained: true`:
> ERROR: Reveal plugin 'RevealChalkboard is not compatible with self-contained output

Maybe a warning is in order, or at least some info in the documentation about what to expect with `print-pdf`?
